channel-manager-api.md

```diff
@@ -139,7 +139,119 @@ Mews automatically sends changes in Inventory \(once connection is set up\). Dat
 
 \[`sync`\] This method is used when Mews updates restrictions.
 
-#### Request `[ChannelManagerApiAddress]/updateRestrictions`
+#### Request Examples `[ChannelManagerApiAddress]/updateRestrictions`
+
+**Stop Sell Restrictions**
+
+- Stop Sell with no exceptions
+
+```javascript
+{
+   "clientToken":"[Mews Client token]",
```

Comment:
is this format of the json which is used everywhere? it is incorrectly formatted. (spaces missing e.g)